busybox/busybox.py

```diff
@@ -9,39 +9,64 @@
 
 from trunner.ctx import TestContext
 from trunner.dut import Dut
-from trunner.types import TestResult, TestSubResult, Status
+from trunner.types import TestResult, Status
+
+
+EXAMPLE_INPUT = """
+---Testcase 'busybox as unknown name' starting---
+echo -ne '' >input
+echo -ne '' | ./unknown 2>&1
+PASS: busybox as unknown name
+---Testcase 'date-@-works' starting---
+FAIL: date-@-works
```

Comment:
I think that it would be better to place it in a comment.

Reply:
I based on the previous comment:
![image](https://github.com/phoenix-rtos/phoenix-rtos-tests/assets/116297248/ed591285-faa5-4567-b5b6-7451d379836d)

Reply:
Please keep it as a multiline variable (reasoning provided previously).

Reply:
Oh, right. Ok, I was worried about some potential issues (flake and so on) when we don't use those variables in fact.